src/styles/blocks/catalog.scss

```diff
@@ -0,0 +1,32 @@
+.catalog {
+  display: grid;
+  grid-template-columns: repeat(1, $card-width);
+  gap: $card-gap;
+  justify-content: center;
+
+  &__card {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+
+    height: $card-height;
+    border: 1px solid #ccc;
+    border-radius: 5px;
+  }
+}
+
+@media (min-width: 488px) {
+  .catalog {
+    grid-template-columns: repeat(2, $card-width);
+  }
+}
```

Comment:
You can use it inside catalog without .catalog {} :)

Reply:
Thank you very much for the tip!

Reply:
let's do it